### PR TITLE
docs(artifact-format): Highlight that the compression is optional

### PR DESCRIPTION
**Pull request by oleorhagen**
Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

File: Documentation/artifact-format-v3.md [hunk 1]
```diff
@@ -487,7 +487,12 @@ tar, this is important.
 Compression
 ===========
 
-All file tree components ending in `.gz` in the tree displayed above should be
+Compression of the Artifact is optional.
```
Comment (TheYoctoJester) [inline Documentation/artifact-format-v3.md:490]: This says "optional", but then is not clear what suffix would apply if it is uncompressed. None at all?

Comment (oleorhagen) [reply Documentation/artifact-format-v3.md:490]: Hmm... Yeh, I get your point. I figured writing it it was implicit ofc, but if you are unsure whilst reading it it should probably be added.

Up to you @TheYoctoJester

Comment (TheYoctoJester) [reply Documentation/artifact-format-v3.md:490]: It should. Want me to do so?

Comment (oleorhagen) [reply Documentation/artifact-format-v3.md]: Done :) Have a look!

**Review by TheYoctoJester — APPROVED** [review]

**Review by kacf — APPROVED** [review]

**Pull request merged**